src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -1,4 +1,53 @@
 package nl.novi.techiteasy.controllers;
+import nl.novi.techiteasy.exceptions.NameNotApprovedException;
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+@RequestMapping("/televisions")
 
 public class TelevisionController {
+    private ArrayList<String> televisionDatabase = new ArrayList<>();
+
+    @GetMapping("/showTv")
+    public ResponseEntity<Object> showTvList() {
+        return ResponseEntity.ok("Lijst van alle tv's: " + televisionDatabase);
+    }
+
+    @PostMapping("/addTv")
```

Comment:
Een PostMapping zegt eigenlijk al /addTv, dus hoef je het er niet nog achter te zetten. Conventie is daarom dus ook een POST request naar /televisions maakt een tv aan